### DISCUSSION TITLE
The attention method is the same than AUTO's?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [MoonRide303/Fooocus-MRE#132] [Q&A] [answered]
Let's say this prompt is correct to use with Fooocus?:

`Masterpiece composition, realistic depiction of a robot holding an old wooden chest, full scene, (glowing eyes), (centered:1.5), (front side view:1.4), realistic skin, unreal engine 5, photoshop, (dramatic lighting:1.2), (Detail Extractor:1.1), sharp details, art by artgerm and greg rutkowski`

**Comment by barepixels** [comment, marked as answer]
(yes :1.4)
